rust/psibase/src/package.rs

```diff
@@ -639,6 +639,32 @@ impl DirectoryRegistry {
     pub fn new(dir: PathBuf) -> Self {
         DirectoryRegistry { dir }
     }
+
+    // TODO: extract sync methods to a proper TestDirectoryRegistry?
```

Comment:
Another option is to provide only the async version and use https://docs.rs/futures/latest/futures/executor/fn.block_on.html for synchronous calls.

Reply:
Beautiful! That worked!!